BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
+
+1. [Grunt Tasks](#grunt-tasks)
+
+## Grunt Tasks
+### clean
+The clean task cleans out the Distribution directory in the FullScreenShenanigans project. This occurs because if a new build has changes made, the Distribution folder must be recreated to reflect those changes.
```

Comment:
> The clean task
> The tslint task

You're starting all task descriptions with the same sentence. Try shortening the phrase and introducing variations and/or alternatives.